fatsecret/fatsecret.py

```diff
@@ -287,12 +295,18 @@ def foods_search(self, search_expression, page_number=None, max_results=None):
         :type page_number: int
         :param max_results: total results per page (default 20)
         :type max_results: int
+        :param region: 2 character ISO country code (default US)
+        :type max_results: str
+        :param language: language code from http://platform.fatsecret.com/api/Default.aspx?screen=rapil#complete_list_of_languages
+        :type max_results: str
         """
         params = {'method': 'foods.search', 'search_expression': search_expression, 'format': 'json'}
 
-        if page_number and max_results:
+        if page_number and max_results and region and language:
```

Comment:
this is super delayed, but I think the localization should be handled separately from the pagination. That way this can be released as a non-breaking change